src/components/SidePanel.tsx

```diff
@@ -1,16 +1,45 @@
-import * as React from 'react';
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { Checkbox, Tooltip } from 'antd';
+
+import { ActiveState, TOOLTIP_COLOR } from '../constants';
+import { VisibilityContext } from '../AgentVisibilityContext';
 
 import '../../css/side_panel.css';
 
-// TODO placeholder, side panel will receive many props when rendering checkboxes
-interface SidePanelProps {}
+const SidePanel: React.FunctionComponent = (): JSX.Element => {
+  const { setAllAgentVisibility: setAllAgentVisibility, allAgentsHiddenState } =
```

Comment:
why is `setAllAgentVisibility` renamed to `setAllAgentVisibility` here?

Reply:
mistake left in during a refactor/rename, sorry!